streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkAppHttpWatcher.java

```diff
@@ -575,6 +578,7 @@ and the status is not obtained this time (flink rest server is closed),
           if (trackingUrl != null && !trackingUrl.equals(application.getJobManagerUrl())) {
             application.setJobManagerUrl(trackingUrl);
             applicationService.updateJobManagerUrl(application.getId(), trackingUrl);
+            applicationLogService.updateJobManagerUrl(application.getId(), trackingUrl);
```

Comment:
Thanks for your contribution. I think the job history record should not be modified. It is just an objective record of the job. The job instance recorded in this table may have stopped running, so it is unreasonable to modify its information.

Reply:
1. We only need to update the jobmanager url while the job is still running, not killed and cancelled states
2. The job history interface is displayed to use the jobmanagerurl, if you do not update this place to show link errors.

Reply:
> 1. We only need to update the jobmanager url while the job is still running, not killed and cancelled states
> 2. The job history interface is displayed to use the jobmanagerurl, if you do not update this place to show link errors.

In the t_flink_log table, an application (appid) may be inserted multiple times. Each operation (app build, app start, app stop, app release) will record information about this app. This PR will modify all records of the app, which will result in logical errors.

Reply:
oh, It is modified according to the yarn appid. Then there is no problem